lib/models/instance.js

```diff
@@ -139,6 +138,7 @@ Instance.prototype.listenToSocketEvents = function () {
  * @return {[type]}      [description]
  */
 Instance.prototype.handleSocketData = function (data) {
+  /*jshint maxcomplexity:12 */
```

Comment:
I'm fairly certain that by adding this, you change the value of it for the remainder of the file... since this is annoying to maintain, maybe we just consider moving this to `semistandard` (because browser)?